### ISSUE TITLE
Duplicate keypresses

### ISSUE THREAD

**Issue by loiclacombe** [UltimateHackingKeyboard/firmware#317]
I have been getting random duplicate keypresses for a few days on Windows 10 20H2. To reproduce, I just have to type in any text field.
I am using firmware version 8.10.2, but I had the saame problem with the previous version in Agent

**Comment by mondalaci** [comment]
Please see https://ultimatehackingkeyboard.com/knowledgebase?q=duplicate and let us know if the issue persists.

**Comment by loiclacombe** [comment]
I tested when the two parts of the keyboard were joined, and it worked right away. I could not reproduce the issue. Then I split back the keyboard and got some more duplicated key input issues.
Then. I rolled back to firmware 8.9.3, with split keyboard and I am not getting any issues anymore. It could be just a random correlation if it is a connectivity/mechanocal issue. So I will tell you if any funky issue arises again.
